guacamole/src/main/webapp/translations/fr.json

```diff
@@ -4,6 +4,9 @@
 
     "APP" : {
 
+        "NAME"    : "Apache Guacamole",
+        "VERSION" : "${project.version}",
```

Comment:
These two lines should *not* be included in the French translation file.

Reply:
Keys removed.